main.tf

```diff
@@ -10,6 +10,8 @@ module "label" {
   enabled     = var.enabled
 }
 
+data "aws_partition" "current" {}
+
```

Comment:
please add 

```
count = var.enabled ? 1 : 0
```

so when the module is disabled, it will not execute the data source